### PR TITLE
travel: rewrite into err

### PR DESCRIPTION
**Pull request by Adman**
Signed-off-by: Adrian Matejov a.matejov@centrum.sk

### CODE REVIEW

**Comment by Adman** [comment]
cc @pepol

File: err-travel/travel.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+        body = msg.body
+
+        if ('zajtra' in body or 'pozajtra' in body) and date is not '':
+            if 'pozajtra' in body:
+                delta = 2
+            elif 'zajtra' in body:
+                delta = 1
+            date = (datetime.date.today() + datetime.timedelta(days=delta)) \
+                .strftime("%d.%m.%Y")
```
Comment (pepol) [inline err-travel/travel.py]: You should be consistent in using single or double quoted throughout the file. Please change these into apostrophes (single quotes).

File: err-travel/travel.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+        vehicle = 'vlakbus'
+        if 'bus' in body:
+            vehicle = "bus"
```
Comment (pepol) [inline err-travel/travel.py]: Single quotes please.

File: err-travel/travel.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+            return 'Not enough arguments specified. See !help for usage'
+
+        dep = args[0]
+        dest = args[1]
+
+        time = args[2] if len(args) > 2 else ''
+        date = args[3] if len(args) > 3 else ''
+
+        if dep == dest:
+            return "You joker"
```
Comment (pepol) [inline err-travel/travel.py]: Single quotes please.

**Comment by pepol** [comment]
@Adman just couple of PEP8 problems, otherwise LGTM!

**Comment by Adman** [comment]
@pepol should be fixed

**Comment by mrshu** [comment]
@Adman provided this works, I am OK with it being merged.

I'd like to see it deployed somewhere soon too.

If I could have one wish, I'd very appreciate if there were some tests for this (as err supports them). That would make me personally much more confident when hitting the big green button.

**Pull request merged**